lib/msal-core/src/telemetry/HttpEvent.ts

```diff
@@ -0,0 +1,73 @@
+import TelemetryEvent from "./TelemetryEvent";
+import { EVENT_NAME_PREFIX } from "./TelemetryConstants";
+import { QPDict } from "../AuthenticationParameters";
```

Comment:
This will change with PR #804. Maybe StringDict?

Reply:
removing